providers/ethereum-provider/rollup.config.js

```diff
@@ -1,10 +1,38 @@
 import { name, dependencies, peerDependencies } from "./package.json";
-import createConfig from "../../rollup.config";
-// @walletconnect/modal has dynamic imports, so we need to enable inlineDynamicImports
-export default createConfig(
-  name,
-  Object.keys({ ...dependencies, ...peerDependencies }),
-  { inlineDynamicImports: true },
-  { inlineDynamicImports: true },
-  { inlineDynamicImports: true },
+import createConfig, { input, plugins } from "../../rollup.config";
+import alias from "@rollup/plugin-alias";
+import path from "path";
+
+const options = { inlineDynamicImports: true };
+
+// keep `@reown/appkit/core` in the external dependencies, else the builds will balloon in size
+const externalDependencies = Object.keys({ ...dependencies, ...peerDependencies }).concat(
```

Comment:
What impact does this have for web builds?

Reply:
none, setting all deps as `external` has always been the case but appkit was incorrectly included because the package is called `@reown/appkit` while the import internally was `@reown/appkit/core` thus being bundled